js/src/components/google-combo-account-card/account-details.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import useGoogleAccount from '.~/hooks/useGoogleAccount';
+import useGoogleAdsAccount from '.~/hooks/useGoogleAdsAccount';
+import useGoogleMCAccount from '.~/hooks/useGoogleMCAccount';
+
+/**
+ * Account details.
+ * @return {JSX.Element} JSX markup.
+ */
+const AccountDetails = () => {
+	const { google } = useGoogleAccount();
+	const { googleAdsAccount } = useGoogleAdsAccount();
+	const { googleMCAccount } = useGoogleMCAccount();
+
+	return (
+		<>
+			<p>{ google.email }</p>
+			<p>
```

Comment:
Since the scope of this issue is only to show the connected accounts and we're handling the reclaim flow in #2604, I think it would best to address these updates as a part of that PR instead. That said, it does look like the [Figma designs](https://www.figma.com/design/fqR0EHi63lWahRcVTKCcba/Google-Listings-%26-Ads-v2.x?node-id=7720-6718&t=LkEXAHSKXDsLiwva-4)show that the MC ID is only shown once the URL is claimed.